ovve_ui/ovve_ui.py

```diff
@@ -546,14 +505,25 @@ def main() -> None:
                         "--sim",
                         action='store_true',
                         help='Run with simulated data')
+
+    parser.add_argument('-w',
+                        "--windowed",
+                        action='store_true',
+                        help='Run in windowed mode (fullscreen default)')
+
+    parser.add_argument('-d',
+                        "--dev_mode",
+                        action='store_true',
+                        help='Run in developer mode (alarm hotkeys enabled, toggle fullscreen enabled)')
+
     parser.add_argument('-p',
                         "--port",
                         default='/dev/ttyUSB0',
                         help='Serial port for communication with MCU')
     args = parser.parse_args()
 
     app = QApplication(sys.argv)
-    window = MainWindow(args.sim)
+    window = MainWindow(args.port, args.sim, not args.windowed, args.dev_mode)
```

Comment:
Passing the port looks OK in develop.  Maybe a bad merge into your branch?
Passing a negative for windowed is confusing.  How about making the option -f for fullscreen?